### PR TITLE
Utility to filter and display if any image in the data directory could be child's full essay.

### PR DESCRIPTION
**Pull request by singparvi**
Description for this pull request --> This pull request is to request the addition of a utility in the util folder that checks and displays all image files that could potentially be a child's full essay. This utility can be used in addition to the already existing utility that checks if an image is a potential child's full essay based on image size. The utility allows for flexibility in setting the directory and setting different image threshold heights (the basis on which the images are filtered). 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

[Trello Card](https://trello.com/c/hOh48vsE/478-clean-up-full-scan-essay-images-in-the-ds-repo)
[Loom Video](https://www.loom.com/share/6093b48e1d504b75998f1ec1f41c43bd)

### CODE REVIEW

**Comment by techthumb1** [comment]
I feel that this will be a very helpful, and timesaving utility, that all developers can benefit from. Easy to follow, and nicely documented. I can very easily reproduce the code following along with the README, and implement the function sufficiently. Great job!

**Review by abdishifa234 — APPROVED** [review]
Parvi it is a nice script better than the previous scripts, one I would like to ask is if I am correct the script only counts the number of full essay images and snippets, I wish the scripts remove or say something about the images.

File: utils/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+1. Files that are likely to be full essays 
+2. Files that are likely to be individual snippets
+
+**What the Script Does Not Do:**
+
+The script does not remove the files that it finds. The user will need to make a determination about whether a file was inadvertently pushed and needs to be removed to comply with COPPA regulations.
+
+The threshold for determining whether or not a file is likely to be a snippet or a full original essay currently is 
+determined by its file size. Any files greater than 260000 bytes are possibly full essay images.
+
```
Comment (BritVandy) [inline utils/README.md:31]: This is explained very well. I like that you included what limitations your script has.

File: utils/essaycheck_w_image_height.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+                    # Get image dimensions
+                    image = Image.open(file_to_check)
+                    width, height = image.size
+
+                    # Check if the image height is more than the max_height
+                    # if it is then print file name:height in console and display to the user
+                    if height > max_height:
+                        print('Potential Full Essay Information (file:height shown) -->', file_to_check, height)
+                        # Display the image to the user
```
Comment (BritVandy) [inline utils/essaycheck_w_image_height.py:38]: Printing the image here is a good idea to fully be able to assess the image. Good job!

**Review by BritVandy — APPROVED** [review]
This script will no doubt have a positive impact on the overall project.

File: utils/README.md (hunk 1 above)
Comment (singparvi) [reply utils/README.md:31]: Hi @BritVandy just to be clear this is not my work but @cryptobellum 's. I think her code is pending merge and that is why it shows as if this is my work.

Comment (cryptobellum) [reply utils/README.md:31]: Correct. My PR was submitted, and I think Parvi used the same branch, so his PR had two commits, one of which was mine. @BritVandy

**Comment by singparvi** [comment]
> Parvi it is a nice script better than the previous scripts, one I would like to ask is if I am correct the script only counts the number of full essay images and snippets, I wish the scripts remove or say something about the images.

The script did show the flagged images and print their height in the console if that is what you are looking for. I am unable to understand the feature you are requesting.

**Pull request merged**